src/components/ParticipantCard.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export default function ParticipantCard({error, loading, name, business_type, address, live_date, precinct}) {
```

Comment:
These props aren't used "anymore" in the component. Please remove 🙂

`{error, loading}`